WebApp/src/components/RegisterPage/RegisterPage.js

```diff
@@ -1,65 +1,220 @@
 // TODO: remove this "eslint-disable" after you remove all console logs...
 /* eslint-disable no-console */
+
 import React, { Component } from 'react';
+import classNames from 'classnames';
 import axios from 'axios';
-import { bind } from '../../util/stateBinding';
+import './RegisterPage.scss';
+import FormValidator from '../FormValidator/FormValidator';
+import PropTypes from 'prop-types';
 
 export default class RegisterPage extends Component {
-  state = {
-    email: '',
-    password: '',
-    passwordCheck: ''
-  };
+  constructor(props) {
```

Comment:
No need for ctor. State & validator can be defined outside of it.

Reply:
later